src/open_telemetry_exporter.cc

```diff
@@ -156,10 +145,12 @@ static void ot_callback_decref(void *data) {
 
   cb->stub.reset();
 
-  sfree(cb->name);
   sfree(cb->host);
   sfree(cb->port);
```

Comment:
Please NULL these too.

Reply:
`sfree()` is a macro that also NULLs the variable.